mobile/lib/pages/recordings_page.dart

```diff
@@ -36,26 +36,57 @@ class RecordingsPage extends HookWidget {
     final recordings = await fetchData(pageUrl, RecordingInfo.fromJson);
 
     final pageData = <RecordingInfo>[];
-    for (final element in recordings) {
-      final (recordingPath, duration) = await fetchSingle(
-        _singleRecordingUrl(element.recordingPath),
-        (jsonData) => (
-          jsonData['source_url'] as String,
-          Duration(seconds: jsonData['media_details']['length'] as int)
-        ),
-      );
-      final thumbnailPath = element.thumbnailPath.isEmpty ? '' : await fetchSingle(
-        _singleRecordingUrl(element.thumbnailPath),
-            (jsonData) => jsonData['media_details']['sizes']['thumbnail']
-        ['source_url'] as String,
-      );
 
+    final Map<String, RecordingDetails> recordingDetails = {
```

Comment:
Dart linter is screaming that the explicit type declaration of local variable is not needed, so either remove it or, if you think it's neccessery (i don't think so) add a comment/a line in `analysis_options.yaml` disabling this lint